cadt/templates/cadt.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=CADT for {{ user }}
+Requires=chia-data-layer.service
+After=chia-data-layer.service
+Requires=chia-wallet.service
+After=chia-wallet.service
+
+[Service]
+ExecStart=/home/{{ user }}/cadt/linux-x64/climate-warehouse
```

Comment:
I'd probably add a definition for CHIA_ROOT in here, even though we're using the default now, so that it works easily in the future as well.

Reply:
good idea - added with recent commit